src/platform_impl/macos/appkit/event.rs

```diff
@@ -212,6 +213,7 @@ unsafe impl Encode for NSEventModifierFlags {
 }
 
 bitflags! {
+    #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
```

Comment:
Could you change the order the way it's written? Usually it's `Debug, Clone, Copy, PartialEq, Eq, Hash` The only justified change is `Clone, Copy` and `PartialEq, Eq` when it comes to ordering, because they of `Lesser -> More` order and LSP inserts both of them that way.

```suggestion
    #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
```

Reply:
> Usually it's `Debug, Clone, Copy, PartialEq, Eq, Hash`

Where does `Default` fit in that order?
> [..], because they of `Lesser -> More` order and LSP inserts both of them that way.

Whats `Lesser` and `More`? What's LSP? Do you mean Rust Analyzer?

Reply:
Doesn't really matter for default tbh.

Reply:
>Whats Lesser and More? What's LSP? Do you mean Rust Analyzer?

Lesser -> more is about semantics they give, `Copy` is extension of `Clone`, And `Eq` is extension of `PartialEq` if you look at them.

`LSP` is a language server protocol, so rust-analayzer in that case.